1-js/02-first-steps/11-logical-operators/9-check-login/solution.md

```diff
@@ -1,25 +1,21 @@
-
-
 ```js run demo
-let userName = prompt("Who's there?", '');
-
-if (userName === 'Admin') {
-
-  let pass = prompt('Password?', '');
+let userName = prompt("Who's there?", "");
 
-  if (pass === 'TheMaster') {
-    alert( 'Welcome!' );
-  } else if (pass === '' || pass === null) {
-    alert( 'Canceled' );
-  } else {
-    alert( 'Wrong password' );
-  }
+if (userName === "Admin") {
+	let pass = prompt("Password?", "");
 
-} else if (userName === '' || userName === null) {
-  alert( 'Canceled' );
+	if (pass === "TheMaster") {
+		alert("Welcome!");
+	} else if (pass === "" || pass === null) {
+		alert("Canceled");
+	} else {
+		alert("Wrong password");
+	}
+} else if (userName === "" || userName === null) {
+	alert("Canceled");
 } else {
-  alert( "I don't know you" );
+	alert("I don't know you");
 }
 ```
```

Comment:
Descrierile din `alert ( 'descriere' )` pot fi traduse. Aici ar fi o idee bună să verifici dacă ai prettier disabled. Codul a fost formatat greșit.

Aici este codul original (care la rândul lui nu e consistent în formatare, recomand să îl lași cum e sau să deschizi un pool request în `en`):

```suggestion


```js run demo
let userName = prompt("Who's there?", '');

if (userName === 'Admin') {

  let pass = prompt('Password?', '');

  if (pass === 'TheMaster') {
    alert( 'Welcome!' );
  } else if (pass === '' || pass === null) {
    alert( 'Canceled' );
  } else {
    alert( 'Wrong password' );
  }

} else if (userName === '' || userName === null) {
  alert( 'Canceled' );
} else {
  alert( "I don't know you" );
}
```

```